### PR TITLE
Travis fixes

### PR DESCRIPTION
**Pull request by stefson**
let's do a testrun

### CODE REVIEW

**Comment by cnd** [comment]
how do you put status `Draft` on PR?

**Comment by stefson** [comment]
@cnd how can I say to use either `MIT` or `Apache-2.0` as a license in https://github.com/gentoo/gentoo-rust/pull/408/commits/81f5a726ff22359afa4483b9ba685eddcc5e1b54? I think right now it says that both are mandantory?

**Comment by cnd** [comment]
@stefson `LICENSE="|| ( foo bar )"`

File: dev-util/xargo/xargo-0.3.12.ebuild [hunk 1]
```diff
@@ -46,7 +46,7 @@ DESCRIPTION="The sysroot manager that lets you build and customize std"
 HOMEPAGE="https://github.com/japaric/xargo"
 SRC_URI="$(cargo_crate_uris ${CRATES})"
 RESTRICT="mirror"
-LICENSE="MIT Apache-2.0" # Update to proper Gentoo format
+LICENSE="|| ( MIT Apache-2.0 )" # Update to proper Gentoo format
```
Comment (cnd) [inline dev-util/xargo/xargo-0.3.12.ebuild]: I guess you can drop # comment too

Comment (stefson) [reply dev-util/xargo/xargo-0.3.12.ebuild]: yeah, I'll clean them up in the end.

**Comment by stefson** [comment]
Looks good to me now. I mostly added metadata with maintainer-wanted and a short description, but also worked out the license issues. Since not all of them are in the main tree, I added those missing to the overlay. Also I had to remove dev-rust/calculator, since upstream disappeared entirely. 

The last critical error from line 538 to 745 is about something I don't understand. Do you know why this is happening?

**Comment by o01eg** [comment]
What about to update `repoman-travis`?

**Comment by cnd** [comment]
I'll merge it for now, please open issue about rest repoman issues, not fully sure about them atm

**Pull request merged**